iati/resources.py

```diff
@@ -57,88 +61,89 @@
 """The name of a file containing an Organisation Schema."""
 
 
-def get_codelist_paths(version=iati.version.STANDARD_VERSION_ANY):
+@iati.version.decimalise_integer
+@iati.version.allow_possible_version
+def get_codelist_paths(version):
     """Find the paths for all Codelists at the specified version of the Standard.
 
     Args:
-        version (str): The version of the Standard to return the Codelists for. Defaults to iati.version.STANDARD_VERSION_ANY.
+        version (str / int / Decimal / iati.Version): The version of the Standard to return the Codelists for.
```

Comment:
I think we need to be clear that this returns paths to embedded and non-embedded codelists. I had to run it to be sure!

Reply:
https://github.com/IATI/pyIATI/pull/297/commits/b26c2c61e475b6c0187e4f390064a204516836eb